oceanmesh/read_fort14.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+def read_fort14(finame="fort.14", read_bou=False):
+    """
+    Reads an ADCIRC fort.14 mesh file.
+
+    Args:
+        finame (str): Name of the input file. Default is 'fort.14'.
+        read_bou (bool): If True, also reads the boundary data.
+
+    Returns:
+        tuple: EToV, VX, B, opedat, boudat, title
```

Comment:
I would use standard nomenclature like elements, nodes, bathymetry etc. instead of EToV, VX, and B...